catalog/spatial/registry/registry-federation-admin-service-impl/src/main/java/org/codice/ddf/registry/federationadmin/service/impl/RegistryPublicationServiceImpl.java

```diff
@@ -142,7 +142,15 @@ public void update(Metacard metacard) throws FederationAdminException {
                 .collect(Collectors.toCollection(HashSet::new));
 
         if (CollectionUtils.isNotEmpty(locations)) {
-            federationAdminService.updateRegistryEntry(metacard, locations);
+            try {
+                federationAdminService.updateRegistryEntry(metacard, locations);
+            } catch (FederationAdminException e) {
+                //This should not happen often but could occur if the remote registry removed the metacard
```

Comment:
This comment is sufficient for me. The fact it's a rare condition makes this less bothersome code.